bpaingest/projects/omg/ingest.py

```diff
@@ -22,6 +23,8 @@
 logger = make_logger(__name__)
 common_context = [OMGSampleContextual, OMGLibraryContextual]
 
+generaliser = SensitiveDataGeneraliser()
+
```

Comment:
maybe we should have an `OMGBaseMetadata` class and build the generaliser there; otherwise every time we import this module, even if we don't instantiate these classes, all the work of loading the generalisation rules is being done

Reply:
Ok I'll take a look.